src/scores/emerging/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
-This module is currently empty.
+Import the functions from the implementations into the public API.
```

Comment:
I think this line can just be deleted

Reply:
Looks like this suggestion has already been committed into the branch